src/ontology/uberon-odk.yaml

```diff
@@ -64,6 +64,8 @@ import_group:
     - id: bfo
     - id: ncbitaxon
       mirror_from: http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim.owl
+    - id: ncbitaxondisjoints
+      mirror_from: http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim-disjoint-over-in-taxon.owl
```

Comment:
Are we certain we want these in our merged import?

https://github.com/obophenotype/uberon/blob/dc3aa6a28753639eb2b3efe9e433c02ede6c628e/src/ontology/Makefile#L534C1-L537C1

Reply:
Well, we do want the disjointness axioms in the released artefacts, so we have to import them somewhere.

On my machine, `refresh-imports` went from ~9 GB of peak memory consumption without importing `taxslim-disjoint-over-in-taxon.owl` to ~10 GB when importing it, for no noticeable difference in runtime (~4 min). Is that wild?

Reply:
Nope, sounds ok to me! Thanks for checking!